modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,375 @@
+  subjects:
+  - kind: ServiceAccount
+    namespace: my-webhook-namespace
+    name: server
+
+- apiVersion: rbac.authorization.k8s.io/v1  <8>
+  kind: ClusterRole
+  metadata:
+    name: my-cluster-role
+  rules:
+  - apiGroups:
+    - admissionregistration.k8s.io
+    resources:
+    - validatingwebhookconfigurations
+    - mutatingwebhookconfigurations
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - get
+    - list
+    - watch
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: my-cluster-role
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: my-cluster-role
+  subjects:
+  - kind: ServiceAccount
+    namespace: my-webhook-namespace
+    name: server
+----
+<1> Delegates authentication and authorization to the webhook server API.
+<2> Allows the admission webhook server to access cluster resource.
+<3> Points to resources. This example points to the `namespacereservations` resource.
+<4> Enables the aggregated API server to create admission reviews.
+<5> Points to resources. This example points to the `namespacereservations` resource.
+<6> Also enables the admission server to access cluster resource.
+<7> Role binding to read the configuration for terminating authentication.
+<8> Default cluster role and cluster role bindings for an aggregated API server.
+
+. Apply those RBAC rules to the cluster:
++
+----
+$ oc auth reconcile -f rbac.yaml
+----
+
+. Create a YAML file called `webhook-daemonset.yaml` that is used to deploy a webhook as a DaemonSet server in a namespace. Restrict it to master nodes only:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: my-webhook-namespace
+  name: server
+  labels:
+    server: "true"
+spec:
+  selector:
+    matchLabels:
+      server: "true"
+  template:
+    metadata:
+      name: server
+      labels:
+        server: "true"
+    spec:
+      serviceAccountName: server
+      containers:
+      - name: my-webhook-container  <1>
+        image: <image_registry_username>/<image_path>:<tag>  <2>
+        imagePullPolicy: IfNotPresent
+        command:
+        - <container_commands>  <3>
+        ports:
+        - containerPort: 8443 <4>
+        volumeMounts:
+        - mountPath: /var/serving-cert
+          name: serving-cert
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8443 <5>
+            scheme: HTTPS
+      volumes:
+      - name: serving-cert
+        secret:
+          defaultMode: 420
+          secretName: server-serving-cert
+----
+<1> Note that the webhook server may expect a specific container name.
+<2> Points to a webhook server container image. Replace `<image_registry_username>/<image_path>:<tag>` with the appropriate value.
+<3> Specifies webhook container run commands here. Replace `<container_commands>` with the appropriate value.
```

Comment:
Can "here" be removed?

Reply:
Absolutely. I think I originally started the sentence with, "Specify webhook container run commands here" and did not correct the end of the sentence when I changed this to "specifies".